lib/services.js

```diff
@@ -80,6 +82,20 @@ async function startHouseArrestService (udid, opts = {}) {
   return new HouseArrestService(socket);
 }
 
+async function startInstrumentService (udid, opts = {}) {
+  const osVersion = opts.osVersion || await getOSVersion(udid, opts.socket);
+  if (Number(osVersion.split('.')[0]) < 14) {
```

Comment:
use `parseInt(x, 10)` to convert string to an integer
`14` could be moved to constants